testutils/robottestutils/robot_utils.go

```diff
@@ -48,13 +51,13 @@ func NewRobotClient(tb testing.TB, logger logging.Logger, addr string, dur time.
 }
 
 // Connect creates a new grpc.ClientConn server running on localhost:port.
-func Connect(port string) (*grpc.ClientConn, error) {
+func Connect(port int) (*grpc.ClientConn, error) {
```

Comment:
Me creating another helper that took some testing port as input (and having it expect an integer type) caused some impedance with tests that were converting the port to a string to concatenate for creating bind addresses or connection addresses.

I changed those to accept ports as integers -- as that is the lingua franca of a port.